python/res/simulator/simulation_context.py

```diff
@@ -91,11 +79,15 @@ def didRealizationFail(self, iens):
 
 
     def isRealizationQueued(self, iens):
-        return iens in self._run_args
+        try:
```

Comment:
Are you sure we want to do this?

Reply:
we decided not have self._run_args any more (since it's in run_context) - will try to beautify a bit.